### PR TITLE
ci: Add chromium build

### LINKED ISSUE #80: Add chromium build to CI
Avoid issues like this:
https://github.com/KhronosGroup/Vulkan-Utility-Libraries/pull/81

### PR DESCRIPTION
**Pull request by juan-lunarg**
closes #80

### CODE REVIEW

File: scripts/update_deps.py
Comment (juan-lunarg) [inline scripts/update_deps.py:1]: In order to make this script executable I had to run chmod on it
`git update-index --chmod=+x scripts/update_deps.py`

**Comment by juan-lunarg** [comment]
Successful gn ninja build

<details>

```
Done. Made 16 targets from 55 files in 236ms
Generating Ninja Files

Done. Made 16 targets from 55 files in 209ms
Running Ninja Build

ninja: Entering directory `out/Debug'
[1/25] CXX obj/vulkan_layer_settings/layer_settings_manager.o
[2/25] CXX obj/vulkan_layer_settings/vk_layer_settings.o
[3/25] STAMP obj/build/buildflag_header_h.stamp
[4/25] ACTION //build:branding_buildflags(//build/toolchain/linux:clang_x64)
[5/25] STAMP obj/build/branding_buildflags.stamp
[6/25] ACTION //build:chromecast_buildflags(//build/toolchain/linux:clang_x64)
[7/25] STAMP obj/build/chromecast_buildflags.stamp
[8/25] ACTION //build:chromeos_buildflags(//build/toolchain/linux:clang_x64)
[9/25] STAMP obj/build/chromeos_buildflags.stamp
[10/25] STAMP obj/build/config/common_deps.stamp
[11/25] STAMP obj/build/config/executable_deps.stamp
[12/25] STAMP obj/build/config/loadable_module_deps.stamp
[13/25] STAMP obj/build/config/shared_library_deps.stamp
[14/25] STAMP obj/build/config/clang/llvm-symbolizer_data.stamp
[15/25] ACTION //build/config/compiler:compiler_buildflags(//build/toolchain/linux:clang_x64)
[16/25] STAMP obj/build/config/compiler/compiler_buildflags.stamp
[17/25] STAMP obj/build/config/linux/linux.stamp
[18/25] STAMP obj/build/config/posix/posix.stamp
[19/25] STAMP obj/build/config/sanitizers/deps.stamp
[20/25] CXX obj/build/config/sanitizers/options_sources/sanitizer_options.o
[21/25] AR obj/build/config/sanitizers/liboptions_sources.a
[22/25] STAMP obj/external/Vulkan-Headers/vulkan_headers.stamp
[23/25] CXX obj/vulkan_layer_settings/vk_layer_settings_helper.o
[24/25] CXX obj/vulkan_layer_settings/layer_settings_util.o
[25/25] AR obj/libvulkan_layer_settings.a
```

</details>

Note. Header only files aren't tested in the gn build though. We could potentially add a test file that does that only for chromium builds. What do you think @mikes-lunarg ?

**Review by mikes-lunarg — APPROVED** [review]
> Note. Header only files aren't tested in the gn build though. We could potentially add a test file that does that only for chromium builds. What do you think @mikes-lunarg ?

I would be happy to see that added, either with a simple stub that includes the header files, or by adding the test build to BUILD.gn

**Comment by juan-lunarg** [comment]
> I would be happy to see that added, either with a simple stub that includes the header files, or by adding the test build to BUILD.gn

I'll add that in a follow up PR.

**Pull request merged**